package.json

```diff
@@ -78,8 +78,8 @@
     "prettier": "^1.14.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^1.6.0",
-    "@typescript-eslint/parser": "^1.6.0",
+    "@typescript-eslint/eslint-plugin": "^2.3.3",
+    "@typescript-eslint/parser": "^2.3.3",
```

Comment:
Can we make a note of specifically what TS versions our lint config effectively supports?

Reply:
Was this done?